html-aam/index.html

```diff
@@ -9365,6 +9365,132 @@ <h4 id="att-controls">`controls`</h4>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th>
+                <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
+              </th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
+              <td>
+                <div class="general">Not mapped</div>
+              </td>
+            </tr>
+            <tr>
+              <th>Comments</th>
+              <td>The value of this attribute can impact the mappings of its related <a href="#att-commandfor">`commandfor`</a> attribute.</td>
+            </tr>
+          </tbody>
+        </table>
+        <h4 id="att-commandfor">`commandfor`</h4>
+        <table aria-labelledby="att-commandfor">
+          <tbody>
+            <tr>
+              <th>HTML Specification</th>
+              <td>`commandfor`</td>
+            </tr>
+            <tr>
+              <th>Element(s)</th>
+              <td><a data-cite="html/form-elements.html#attr-commandfor">`button`</a>;</td>
+            </tr>
+            <tr>
+              <th>[[WAI-ARIA-1.2]]</th>
+              <td>
+                <div class="general">
+                  <p>
+                    If the associated element is displayed as a popover:
+                    <a class="core-mapping" href="#ariaExpandedTrue">`aria-expanded=true`</a>
+                  </p>
+                  <p>
+                    If the element with the `commandfor` attribute has a `command` attribute with value of "`show-popover`", or "`toggle-popover`" and the associated element is hidden, or not present
+                    in the DOM:
+                    <a class="core-mapping" href="#ariaExpandedFalse">`aria-expanded=false`</a>
+                  </p>
+                  <p>
+                    If the element with the `commandfor` attribute has a `command` attribute with value of "`show-popover`", or "`toggle-popover`", and the associated element is an accessibility
+                    ancestor of the element with the `commandfor` attribute:
+                    <a class="core-mapping" href="#ariaExpandedUndefined">`aria-expanded=undefined`</a>
+                  </p>
```

Comment:
per mention in previous comment, missing expectations for commandfor + command='show-modal'